src/my_files/my_files.rs

```diff
@@ -118,4 +118,7 @@ impl MyFiles {
             Err(error) => Err(error),
         }
     }
+    pub fn raw_query(&self, query: String, params: &[&dyn ToSql]) -> Result<usize> {
```

Comment:
Tell me more about this 

`params: &[&dyn ToSql]`

Reply:
I'll reply to you on discord